test/helpers/fixture_set.rb

```diff
@@ -0,0 +1,76 @@
+require 'helpers/kubeclient_helper'
+
+module FixtureSetAssertions
+  class FixtureSet
```

Comment:
This class is an abstract superclass that cannot be instantiated on its own, and its subclasses each encapsulate knowledge about the contents of a fixture set... they're extremely tied to the fixtures.